content/sensu-enterprise/2.6/guides/using-contact-routing.md

```diff
@@ -0,0 +1,623 @@
+
+The reason for this is that since we have one or more contacts that do not have a configuration override for **email**, we use the default configuration.
+
+![Single Handler with Some Matching Contacts](/images/contact-routing/single-handler/some-matching-contacts.png)
+
+## Multiple Handlers
+
+### Multiple Handlers with a Matching Contact
+
+This configuration we have two handlers, **email** and **slack**, a contact **support** configured. The **support** contact has a configuration override for **email** that will change the default _to_ email address from "default@example.com" to "support@sensuapp.com" and a configuration override for **slack** to change the _channel_ from "#alerts" to "#support"
+
+{{< highlight json >}}
+{
+  "checks": {
+    "example_check": {
+      "command": "do_something.rb",
+      "handlers": [
+        "email",
+        "slack"
+      ],
+      "contacts": [
+        "support"
+      ]
+    }
+  },
+  "contacts": {
+    "support": {
+      "email": {
+        "to": "support@sensuapp.com"
+      },
+      "slack": {
+        "channel": "#support"
+      }
+    }
+  }
+}
+{{< /highlight >}}
+
+Because the contact configuration matches both handlers being used, any event generated for this check will use this contact's configuration.
+
+![Multiple Handlers with Matching Contact](/images/contact-routing/multiple-handlers/matching-contact.png)
+
+### Multiple Handlers with a Single Matching Contact
+
+This configuration we have two handlers **email** and **slack** a contact **support** configured. The **support** contact has a configuration override for only **email** that will change the default _to_ email address from "default@example.com" to "support@sensuapp.com". The **support** contact does not have a handler override for the **slack** handler.
+
+{{< highlight json >}}
+{
+  "checks": {
+    "example_check": {
+      "command": "do_something.rb",
+      "handlers": [
+        "email",
+        "slack"
+      ],
+      "contacts": [
+        "support"
+      ]
+    }
+  },
+  "contacts": {
+    "support": {
+      "email": {
+        "to": "support@sensuapp.com"
+      }
+    }
+  }
+}
+{{< /highlight >}}
+
+Because the contact configuration only has configuration override for the **email** handler, only the **email** handler will be override. The default configuration for **slack** will be used.
+
+![Multiple Handlers with Single Matching Contact](/images/contact-routing/multiple-handlers/single-matching-contact.png)
+
+### Multiple Handlers with a Single Non-Matching Contact
+
+Similar to the previous configuration, we have two handlers, **email** and **slack**, and a contact **support** configured. The **support** contact has a configuration override for **pagerduty** that will change the default _service_key_. The support contact does not have a handler override for the **slack** or **email** handler.
```

Comment:
```suggestion
Similar to the previous configuration, we have two handlers, **email** and **slack**, and a contact **support** configured. The **support** contact has a configuration override for **pagerduty** that will change the default _service\_key_. The support contact does not have a handler override for the **slack** or **email** handler.
```

Reply:
Escapes the second `_` so `service_key` is in italics instead of just `service` and then a `key_` at the end.